ci_helpers/helpers.sh

```diff
@@ -7,17 +7,27 @@ enable_moto_server(){
   moto_server -p5000 2>/dev/null&
 }
 
-create_ftp_server(){
-  docker run -d -p 21:21 -p 21000-21010:21000-21010 -e USERS="user|123|/home/user/dir" -e ADDRESS=localhost --name my-ftp-server delfer/alpine-ftp-server 
+create_ftp_ftps_servers(){
+  docker run -d \
+    --name my-ftp-ftps-servers \
+    --publish 20-21:20-21/tcp \
+    --publish 40000-40009:40000-40009/tcp \
+    --publish 90:90/tcp \
+    --env USER=user \
+    --env PASS=123 \
+    --env HOME_DIR=/home/user \
+    --env FTP_PORT=21 \
+    --env FTPS_PORT=90 \
+    rachitsharma2001/ftp_ftps
```

Comment:
Running external docker images is scary, both from security and also maintenance perspective.

Any wya to launch the FTP(S) server directly from Python here?

Reply:
Thank you for letting me know. I have removed the docker dependency and instead directly set up the ftp and ftps servers on ci using the `vsftpd` linux tool. Let me know if this looks good or if there is anything else I should add or change.